### PR TITLE
Stable20170525

### PR DESCRIPTION
**Pull request by kbrownell**
# READY FOR REVIEW

# Summary
- These commits bring module versions up to the stable20170525 gitolite branch.

# Needed By (Date)
- I don't believe there is a required by date.

# Urgency
- It'd be good to merge this sooner rather than later, just so we can be sure future changes to the deployer are consistent with this state.

# Steps to Test
- See test results below.  Please let me know if you would like a fuller suite of tests run on sites-uat.  Travis used the deploy tag, so that tests can complete within the roughly 50m time period given.

# Associated Issues and/or People
- https://stanfordits.atlassian.net/projects/DEVOPS/issues/DEVOPS-16

We'll also want to review and merge these pull requests for DEVOPS-16.
- https://github.com/SU-SWS/stanford-jumpstart-deployer/pull/145
- https://github.com/SU-SWS/stanford-jumpstart-deployer/pull/144

### CODE REVIEW

**Comment by jbickar** [comment]
Travis passes; I need to manually review the changes.

**Comment by jbickar** [comment]
I've reviewed this manually and made some additions and corrections. Waiting on Travis now.

**Comment by jbickar** [comment]
@sherakama this is GTG from our end. Any suggestions or concerns?

File: development/product/drupalcamp/stanford.make [hunk 1]
```diff
@@ -22,6 +22,12 @@ projects[stanford_bean_types][download][type] = "git"
 projects[stanford_bean_types][download][url] = "git@github.com:SU-SWS/stanford_bean_types.git"
 projects[stanford_bean_types][download][branch] = "7.x-2.x-dev"
 
+projects[stanford_capx][type] = module
```
Comment (sherakama) [inline development/product/drupalcamp/stanford.make:25]: Drupalcamp doesn't need this.

**Review by sherakama — APPROVED** [review]
The Drupal Camp profile doesn't need CAPx but it also doesn't hurt. I suspect you are including it because it is on the 'stack'

File: development/product/drupalcamp/stanford.make (hunk 1 above)
Comment (jbickar) [reply development/product/drupalcamp/stanford.make:25]: Correct; including it because it's on the "stack"

**Pull request merged**